.gitignore

```diff
@@ -16,3 +16,5 @@ tmp/
 # Mac OSX Finder files.
 **/.DS_Store
 .DS_Store
+
+yarn.lock
```

Comment:
There's a simple reasoning: it's not related to the subject of the PR, so it should not be in it. The arguable nature of the change (recommended to commit in docs, with a special case to ignore for you), makes a stronger case to separate the decision from the PR.